addons/default/visiosoft/cats-module/src/Category/Table/CategoryTableBuilder.php

```diff
@@ -86,7 +86,7 @@ class CategoryTableBuilder extends TableBuilder
      * @var array
      */
     protected $options = [
-
+        'table_view' => 'visiosoft.module.cats::table/table'
```

Comment:
it's now working with new pyro @spektra2147

Reply:
yes I'm tested